README.md

```diff
@@ -21,7 +21,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .exec_file(Path::new("/usr/bin/firecracker"))
         .build();
 
-    let root_drive = Drive::builder("root", Path::new("debian.ext4"))
+    let root_drive = Drive::builder("root", Path::new("/tmp/debian.ext4"), Path::new("debian.ext4"))
```

Comment:
user shouldn't need to specify the same paths twice. User should only specify the path and if we're dealing with jailer, we build the host path ourselves.